src/components/tile/tile.component.js

```diff
@@ -75,6 +77,23 @@ Tile.propTypes = {
    * If unset or zero, this will default to 100%.
    */
   width: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  /** Sets the border width by using these design tokens */
+  borderWidth: PropTypes.oneOf([
```

Comment:
We should also update the TypeScript interface for `Tile` in `tile.d.ts` to include these new props.

Reply:
Done